website_base_multi_image/models/image.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#        Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+from openerp import api, models
+
+
+class Image(models.Model):
+    _inherit = "base_multi_image.image"
+
+    @api.model
+    def get_images_to_add(self, post, model_name):
+        """
+        @param post: dict with images in format 'new_image_xxx'
+        @param model_name: Model to store image
+        @return: List of images to add
+        """
+        images_to_add = []
+        for key in post.keys():
```

Comment:
Why not `for key, image in post.iteritems()`. IMO it would make the list check and `new_image` assignment block much cleaner.